tools/Dashboard/DevHome.Dashboard/Views/DashboardView.xaml.cs

```diff
@@ -246,6 +251,24 @@ private async Task RestorePinnedWidgetsAsync(Widget[] hostWidgets)
                     continue;
                 }
 
+                // Ensure only one copy of a widget is pinned if that widget's definition only allows for one instance.
+                var widgetDefinitionId = widget.DefinitionId;
+                var widgetDefinition = await Task.Run(() => catalog?.GetWidgetDefinition(widgetDefinitionId));
+                if (widgetDefinition.AllowMultiple == false)

```

Comment:
Since `widgetDefinition` can be null (`catalog?`)
```suggestion
                if (widgetDefinition?.AllowMultiple == false)
```